examples/04_manipulating_images/plot_extract_regions_labels_image.py

```diff
@@ -0,0 +1,118 @@
+"""
+Regions labeling and extraction using Yeo atlas
+=============================================================
+
+This example shows how to use :class:nilearn.regions.extract_regions_labels_img
```

Comment:
Can you change the above line to this :class:`nilearn.regions.extract_regions_labels_img`. Two marks are missing while referring ``